src/server/utilities/liteATIClickTracking/index.test.ts

```diff
@@ -0,0 +1,186 @@
+import { LITE_ATI_TRACKING } from '#app/hooks/useClickTrackerHandler';
+import liteATIClickTracking from '.';
+
+const createAnchor = ({
+  href = '/',
+  hasliteSiteTracking,
```

Comment:
Let's default this to the most frequently used value, so that we only need to override it in a few test cases.